src/vs/workbench/api/node/mainThreadHandlerRegistry.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+/**
+ * MainThreadHandlerRegistry maintains a collection of objects by handle.
```

Comment:
Don't think that belongs here because this could the theoretically used for other things. I can add doc to the instantiation site though.